jicofo/rootfs/healthcheck.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Don't put these files directly in the root please. /usr/local/bin feels like the right palce for them.

Reply:
done